[2022-03-18 15:22:06,609] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2011-01-09T04:00:00+00:00 [queued]>
[2022-03-18 15:22:06,657] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2011-01-09T04:00:00+00:00 [queued]>
[2022-03-18 15:22:06,667] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 15:22:06,670] {taskinstance.py:1239} INFO - Starting attempt 2 of 3
[2022-03-18 15:22:06,675] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 15:22:06,718] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_pbp_data> on 2011-01-09 04:00:00+00:00
[2022-03-18 15:22:06,760] {standard_task_runner.py:52} INFO - Started process 402 to run task
[2022-03-18 15:22:06,773] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pbp_bucket_ingestion_dag', 'get_pbp_data', 'scheduled__2011-01-09T04:00:00+00:00', '--job-id', '6450', '--raw', '--subdir', 'DAGS_FOLDER/pbp_ingestion_dag.py', '--cfg-path', '/tmp/tmp8w1x2jmf', '--error-file', '/tmp/tmp2vv_k7hp']
[2022-03-18 15:22:06,791] {standard_task_runner.py:77} INFO - Job 6450: Subtask get_pbp_data
[2022-03-18 15:22:07,047] {logging_mixin.py:109} INFO - Running <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2011-01-09T04:00:00+00:00 [running]> on host 9a1698cd207d
[2022-03-18 15:22:07,152] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pbp_bucket_ingestion_dag
AIRFLOW_CTX_TASK_ID=get_pbp_data
AIRFLOW_CTX_EXECUTION_DATE=2011-01-09T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2011-01-09T04:00:00+00:00
[2022-03-18 15:22:07,744] {python.py:175} INFO - Done. Returned value was: None
[2022-03-18 15:22:07,766] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=pbp_bucket_ingestion_dag, task_id=get_pbp_data, execution_date=20110109T040000, start_date=20220318T152206, end_date=20220318T152207
[2022-03-18 15:22:07,849] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-18 15:22:07,931] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
